cmd/skipper/main.go

```diff
@@ -129,6 +128,9 @@ const (
 	enableHopHeadersRemovalUsage         = "enables removal of Hop-Headers according to RFC-2616"
 	oauth2TokeninfoURLUsage              = "sets the default tokeninfo URL to query information about an incoming OAuth2 token in oauth2Tokeninfo filters"
 	maxAuditBodyUsage                    = "sets the max body to read to log inthe audit log body"
+
```

Comment:
nit: hmm separate it from the rest?
If you do this we should probably do it in blocks in a separate PR to make this more readable for all the things.

Reply:
definitely, i just wanted to keep the changeset in this PR small. Is it fine if I just create a separate PR for the formatting?

Reply:
yes